app/Models/Activity.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Model;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Activity extends Model
+{
+    protected $table = 'activities';
+    protected $fillable = [
+        'activity_name',
+        'type',
+    ];
+
+    public function user()
+    {
+        return $this->belongsToMany('App\Models\User');
```

Comment:
return $this->belongsToMany(User::class);
fix tương tự cho những model khác

Reply:
vang ah